types/delaunator/delaunator-tests.ts

```diff
@@ -1,4 +1,4 @@
-import * as Delaunator from 'delaunator';
+import { Delaunator } from 'delaunator';
```

Comment:
Should be

```ts
import Delaunator from 'delaunator';
```

https://github.com/mapbox/delaunator/blob/master/index.js#L5

Also in their tests:

https://github.com/mapbox/delaunator/blob/master/test.js#L3

`import { Delaunator }` would resolve `undefined`